src/views/home/Index.svelte

```diff
@@ -70,14 +70,9 @@
         <div class="project">
           <Link
             route={{
-              resource: "projects",
-              params: {
-                view: { resource: "tree" },
-                id: project.id,
-                baseUrl,
-                peer: undefined,
-                revision: undefined,
-              },
+              resource: "project.tree",
+              project: project.id,
+              seed: baseUrl,
```

Comment:
This could also be your own node which might not be a public seed. Could maybe use `node`?

Reply:
We should maybe discuss this, because on a high level we use peer/node/seed interchangeably. But in the code there is a distinction. In the context of browsing a project we're interested in a node's httpd base url. The node can be either our own (private) or a public seed, but it's always a node. And when selecting a peer in the peer selector, we're talking about the nodeId of the Radicle network peer.

Reply:
Good point d751e7c09a0d8513126d7b8c6cdc4bd9309dcf79